dist/bin/common

```diff
@@ -61,6 +61,14 @@ if [ ! -x "$JAVACMD" ] ; then
   exit 1
 fi
 
+# parses 1.7, 1.8, 9, etc out of java version "1.8.0_91"
+JAVA_VERSION=$("$JAVACMD" -Xmx512M -version 2>&1 | grep ' version "' | sed 's/.*version "\([0-9]*\)\(\.[0-9]*\)\{0,1\}\(.*\)*"/\1\2/; 1q')
```

Comment:
A minor note: why `-Xmx512M`, it seems it's not necessary. But to be safer, let's just keep it as it is.

Reply:
Copy pasted form https://github.com/sbt/sbt-launcher-package/blob/master/src/universal/bin/sbt-launch-lib.bash#L183

Reply:
But I suppose it is not needed here